File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String answer = "";
        
        String [] num = s.split(" ");
        
        int max,min,n;
        min = max = Integer.parseInt(num[0]);
        for(int i = 0 ; i < num.length ; i++){
            n = Integer.parseInt(num[i]);
            if(min > n)min = n;
            if(max < n)max = n;
        }
        answer = min + " " + max;
        return answer;
    }
}

File: C#/프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cs
using System.Linq;
public class Solution {
    public string solution(string s) {
            string answer = "";
            string[] sArr = s.Split(' ');
            int[] nArr = new int[sArr.Length];
            for (int i = 0; i < sArr.Length; i++) 
                nArr[i] = int.Parse(sArr[i]);

            string max = nArr.Max().ToString();
            string min = nArr.Min().ToString();
            answer = string.Join(" ", min, max);
            return answer;
    }
}